package/libretro-cores/libretro-cores.mk

```diff
@@ -0,0 +1,48 @@
+LIBRETRO_CORES_VERSION = origin/master
```

Comment:
Libretro cores are not versioned, like retroarch,

Reply:
Yes that's why ppl use commit sha to pick the best option in that case. Otherwise why we do it if we have daily builds from official buildbot, we could as well wget artifact, check sha265 and extract. Aside from that, theres big bottleneck in package buildtime

Reply:
I will abandon the libretro-super repository and try to build the cores as separate packages